src/screens/Ticketing/Ticket/TicketInfo.tsx

```diff
@@ -133,7 +134,8 @@ const WarningMessage = (
       <ThemeText type="body__secondary">
         {t(
           TicketTexts.warning.anotherMobileAsToken(
-            getDeviceName(inspectableToken),
+            getDeviceName(inspectableToken) ||
```

Comment:
I kinda used `||` on purpose as it is no sanity check on the value anywhere else in the code, and it can be modified outside of our control in clients like Sørvis BackOffice, so in theory it could be an empty string.

Would probably be better to check for empty string in `getDeviceName` and return `undefined` if empty. Or maybe even better to have the `getDeviceName` also handle the fallback to "Unknown device".

Reply:
I was also thinking of having the Unknown device logic in getDeviceName itself as it is now duplicated in a lot of places.